examples/asr/transcribe_speech.py

```diff
@@ -257,7 +266,9 @@ def main(cfg: TranscriptionConfig) -> Union[TranscriptionConfig, List[Hypothesis
 
     # Setup decoding strategy
     if hasattr(asr_model, 'change_decoding_strategy'):
-        if cfg.decoder_type is not None:
+        if isinstance(asr_model.decoding, MultiTaskDecoding):
```

Comment:
I think compute_langs and stuff is supported no ? Oh well

Reply:
CC @stevehuang52

Reply:
yes, I added it. However, it's strange that the users can control this in two ways: `cfg.compute_langs` and `cfg.multitask_decoding.compute_langs`. I overwrote as  `cfg.multitask_decoding.compute_langs=cfg.compute_langs`

Reply:
They should override just config.compute_langs, it's valid is forwarded to the decoding of appropriate model